app/views/data_exports/_new_sample_export_dialog.html.erb

```diff
@@ -62,7 +62,7 @@
             </div>
```

Comment:
The `Samples(NUM selected)` button class `rounded-t` in this file needs to be updated to `rounded-t-lg`.

Reply:
Thanks, updated in f4e5af291db807334c9a2e5f665fa306dc85cba9